build.gradle

```diff
@@ -14,10 +16,21 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib"
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
     version = '2020.3.3'
+    plugins = ['android']
 }
+
+//runIde {
```

Comment:
Can we remove this? Or add a comment of a reason why you would want to uncomment this, such as to test something specific.